app/models/linkset.rb

```diff
@@ -11,11 +11,21 @@ class Linkset < ApplicationRecord
       message: "does not appear to be a valid URL"
   end
 
+  after_commit :verify_linkbacks
+
   def empty?
     LINKS.map { |link| attributes[link] }.all?(&:blank?)
   end
 
   def update_attributes_from_gem_specification!(spec)
     update!(home: spec.homepage)
   end
+
+  def verified?(link)
+    !!self["#{link}_verified_at"]
```

Comment:
if rubocop complains at this like, you could do: 
```suggestion
    !self["#{link}_verified_at"].nil?
```